### PR TITLE
[Experimental] Apache Arrow Torch Serde

### PR DESCRIPTION
**Pull request by tudorcebere**
## Description
Please include a summary of the change, the motivation, and any additional context that will help others understand your PR. If it closes one or more open issues, [please tag them as described here](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [ ] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [ ] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

File: packages/syft/proto/lib/torch/tensor.proto [hunk 1]
```diff
@@ -49,9 +45,24 @@ message TensorData {
...
+    bytes arrow_data = 5;
+  }
+
+  string dtype = 6;
+}
+
+
 message TensorProto {
-  TensorData tensor = 1;
+  bytes tensor = 1;
```
[the remark this answers is not in the document]
Comment (tudorcebere) [reply packages/syft/proto/lib/torch/tensor.proto:64]: No, basically, there lives the whole data hungry part of the tensor and I don't want to get a recursive copy out of it. After I compute it, I turn it into bytes, which has a `memoryview` implementation that makes copying around (in other protobufs) way faster.

File: packages/syft/tests/syft/lib/torchvision/allowlist_test.py [hunk 2]
```diff
@@ -79,6 +82,7 @@ def test_allowlist(
             and version_supported(support_dict=allowlist[item])
         ):
             try:
+                print(item + TEST_PARAMS[item])
```
[the remark this answers is not in the document]
Comment (tudorcebere) [reply packages/syft/tests/syft/lib/torchvision/allowlist_test.py]: Done

File: packages/syft/proto/lib/torch/tensor.proto [hunk 3]
```diff
@@ -4,15 +4,11 @@ package syft.lib.torch;
...
-  string dtype = 2;
 
-  bool is_quantized = 3;
-  float scale = 4;
-  int32 zero_point = 5;
+message ProtobufContent {
+  repeated int64 shape = 1;
 
-  // Field numbers starting at 16 take two bytes to encode,
+    // Field numbers starting at 16 take two bytes to encode,
```
[the remark this answers is not in the document]
Comment (tudorcebere) [reply packages/syft/proto/lib/torch/tensor.proto]: Done

**Review by gmuraru — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**